integrationTests/node/dblookupext/partialMiniblockSave_test.go

```diff
@@ -129,5 +126,5 @@ func TestSavePartialMiniblocksShouldKeepAllData(t *testing.T) {
 	txHash5 := []byte(fmt.Sprintf(txHashPattern, 5)) // hash 5 should not be linked, it was not processed yet
 	mbData, err = historyRepository.GetMiniblockMetadataByTxHash(txHash5)
 	require.Nil(t, mbData)
-	assert.Equal(t, dblookupext.ErrNotFoundInStorage, err)
+	assert.NotNil(t, err)
```

Comment:
Is it possible to assert for a more precise error here?

Reply:
done & simplified the error definition & usage